### ISSUE TITLE
`certs/cert_chain.pem` permissions are wrong in the gem from rubygems

### ISSUE THREAD

**Issue by ojab** [karafka/karafka-testing#114]
```
I have no name!@33477f4db36a:/app$ ls -larth /app/vendor/bundle/ruby/3.1.0/gems/karafka-testing-2.0.6/certs/cert_chain.pem
-rw-------. 1 root root 1.6K Nov  2 09:20 /app/vendor/bundle/ruby/3.1.0/gems/karafka-testing-2.0.6/certs/cert_chain.pem
```
in the docker image using karafka-testing and 
```
Run cp -r /app/vendor/bundle ./vendor/
cp: cannot open '/app/vendor/bundle/ruby/3.1.0/gems/karafka-testing-2.0.6/certs/cert_chain.pem' for reading: Permission denied
Error: Process completed with exit code 1.
```
in CI where gems are installed by root and everything is run by unprivileged user.

`certs/cert_chain.pem` should be `a+r` (and it is in the repo), it's not a security issue in this case anyway.

**Comment by mensfeld** [comment]
It is correct in the repo, let me check what happens after build. If all good, I will close as it will be fixed with the next release.

**Comment by mensfeld** [comment]
Yeah. For any reason it's broken (permission). Let me craft a new release that will fix this.

**Comment by mensfeld** [comment]
Fixed in 2.0.7
